schema.cc

```diff
@@ -1302,6 +1309,11 @@ schema_builder& schema_builder::with_tombstone_gc_options(const tombstone_gc_opt
     return *this;
 }
 
+schema_builder& schema_builder::with_per_partition_rate_limit_options(const db::per_partition_rate_limit_options& opts) {
+    add_extension(db::per_partition_rate_limit_extension::NAME, ::make_shared<db::per_partition_rate_limit_extension>(opts));
```

Comment:
How does it work in mixed clusters? Suppose we call this on an upgraded node, what will an old node think of the new extension?

Perhaps we need to restrict this until the cluster is fully updated.